### DISCUSSION TITLE
Ente Mobile/Desktop App Passkey & Cast Support - Selfhosted Instances

### DISCUSSION THREAD

**Discussion by vicky-bs** [ente-io/ente#3777] [Enhancements]
On selfhosted ente instance (server, web, albums, accounts) where a passkey is successfully created on ente web, user will not be able to login to mobile app as the passkey verification is routed to accounts.ente.io (hardcoded) instead of accounts.example.com (selfhosted). Would be great to get the changes implemented in the app that will redirect to accounts.example.com for passkey verification

Same happens when it comes to casting (using mobile/desktop). An option should be provided in setting to choose customer endpoint for both

**Comment by ekadin-mtc** [comment]
This applies to the desktop app as well...

**Comment by ua741** [comment]
You can now configure these urls in your server config. [Ref](https://github.com/ente-io/ente/blob/543411254b2bb55bd00a0e515dcafa12d12d3b35/server/configurations/local.yaml#L76-L89).
Clients that should honor these configuration will be soon available across all platforms (mobile photos: v0.9.75, auth: v4.2.3)

**Reply by vicky-bs** [reply]
@ua741 Thank you. Maybe the documentation can be a little more clearer on the format of the yaml file

Should the format in local.yaml be

public-albums: https://albums.example.com

Or 

https://albums.example.com
public-albums:

Or 

public-albums:
                          https://albums.example.com

**Reply by ekadin-mtc** [reply]
The first option works
